delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/eventlog/ExpandedGlobalEventLog.scala

```diff
@@ -25,29 +27,39 @@ final class ExpandedGlobalEventLog private (
 
```

Comment:
The class `ExpandedGlobalEventLog.scala` should be part of the blazegraph plugin, not of the core. This is only used there